packages/build-info/src/frameworks/nuxt.ts

```diff
@@ -30,13 +30,13 @@ export class Nuxt extends BaseFramework implements Framework {
       if (this.isV3(this.detected)) {
         this.name = 'Nuxt 3'
 
-        const cmd = this.project.packageManager?.runCommand || 'npm run'
-        this.build.command = `${cmd} build`
-        this.dev.command = `${cmd} dev`
+        this.build.command = `nuxt build`
```

Comment:
```suggestion
        this.build.command = `nuxi build`
```

Reply:
One of the biggest changes to come with Nuxt 3 is the `nuxi` CLI that leverages [Nitro Server](https://nitro.unjs.io/). So all commands are `nuxi` instead of `nuxt`.

Reply:
@bencodezen since when is the `nuxi` cli shipped? just created a site yesterday and the commands that where generated still where `nuxt`

![CleanShot 2023-04-27 at 15 52 54](https://user-images.githubusercontent.com/11156362/234883697-2703b275-1004-423a-b9b0-55ce627c96da.png)

Reply:
You're right. It looks like it got reverted. So we're good now!